tests/e2e/specs/page-content-runtime.test.ts

```diff
@@ -0,0 +1,28 @@
+describe('Webextension Content Runtime Script', () => {
+  before(function () {
+    if ((browser.capabilities as WebdriverIO.Capabilities).browserName === 'chrome') {
+      // Chrome doesn't allow content scripts on the extension pages
+      this.skip();
+    }
+  });
```

Comment:
Why not to open

```ts
await browser.url('https://www.example.com');
```

like in `content-ui`?

Reply:
There is no way to open popup from actual browser menu and click `Inject Content Script` button using BiDi protocol on it.

There were some complicated workarounds using Chrome-Dev Protocol which may work. But for now all popup tests are done using directly html page of the popup and chrome somehow restricted it.

Reply:
@govza Ok, keep it in mind, to refactor it in the future, if it's possible :)